crates/uv-platform-tags/src/platform_tag.rs

```diff
@@ -72,8 +72,8 @@ pub enum PlatformTag {
 }
 
 impl PlatformTag {
-    /// Returns `true` if the platform is manylinux-compatible.
-    pub fn is_manylinux_compatible(&self) -> bool {
```

Comment:
I renamed these, since, e.g., the `Self::Any` tag is also compatible, but it doesn't get at what we want here.